thirdeye/thirdeye-frontend/app/mirage/config.js

```diff
@@ -100,6 +80,63 @@ export default function() {
           })
         }
       }
-    }
+    };
+  });
+
+  /**
+   * Mocks a list of alerts, displayed in the /manage/alerts page
+   */
+  this.get('/thirdeye/entity/ANOMALY_FUNCTION', () => {
+    return anomalyFunction;
+  });
+
+  /**
+   * Mocks email subscription groups for alerts
+   */
+  this.get('/thirdeye/entity/ALERT_CONFIG', () => {
+    return alertConfig;
+  });
+
+  /**
+   * Mocks a list of applications that are onboarded onto ThirdEye
+   */
+  this.get('/thirdeye/entity/APPLICATION', () => {
+    return entityApplication;
+  });
+
+  /**
+   * Returns information about the first alert in anomalyFunction mock data
+   */
+  this.get(`/onboard/function/${anomalyFunction[0].id}`, () => {
+    return anomalyFunction[0];
+  });
+
+  /**
+   * Returns metric information about the first alert
+   */
+  this.get('/data/autocomplete/***', () => {
+    return metric;
+  });
+
+  /**
+   * Returns max data time.
+   */
+  this.get(`/data/maxDataTime/metricId/${metric[0].id}`, () => {
+    return 1509051599998;
+  });
+
+  /**
+   * Returns the first email config
+   */
+  this.get(`/thirdeye/email/function/${anomalyFunction[0].id}`, () => {
+    return alertConfig[0];
+  });
+
+  /**
+   * Returns mock data to render time series graph
+   */
+  this.get(`/timeseries/compare/${metric[0].id}/1506459599999/1509051599999/
+  1505854799999/1508446799999?dimension=All&granularity=1_HOURS&filters=%7B%7D`, () => {
```

Comment:
Let's set the query params as *** for now since we're not using them

Reply:
so change it to `/timeseries/compare/***`?

Reply:
perhaps this instead `/timeseries/compare/${metric[0].id}/***` ?